sklearn/multiclass.py

```diff
@@ -511,7 +512,7 @@ def fit(self, X, y):
         return self
 
     def predict(self, X):
-        """Predict multi-class targets using underlying estimators.
+        """Returns argmax(decision_function(X), axis=1)
```

Comment:
I prefer to be more explicit:

def predict(self, X):
    """Estimate the best class label for each sample in X

    This is implemented as ``argmax(decision_function(X), axis=1)`` which
    will return the label of the class with most votes by estimators predicting
    the outcome of a decision for each possible class pair.

    """